packages/gatsby-cli/package.json

```diff
@@ -46,7 +46,11 @@
   ],
   "license": "MIT",
   "main": "lib/index.js",
-  "repository": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-cli",
```

Comment:
So as @wardpeet noted, this is something we'll want to do (eventually?), but I do think this is _maybe_ a worse experience in two scenarios:

1. Browsing on npmjs.org and using the repository link, e.g. [gatsby-cli](https://www.npmjs.com/package/gatsby-cli)
1. Using `npm repo <repo-name>`, e.g. `npm repo gatsby-cli`

In both of these scenarios, the root of the repo will open, rather than the specific package.

Reply:
Agreed - until npm improves their links to support the new format properly the hack is probably a better experience.

Maybe worth leaving this open, but with a comment that support for https://github.com/npm/cli/pull/140, and an implementation of the npmjs.org change (I believe that repo is private), would be awesome.